src/gridfs/download.ts

```diff
@@ -1,14 +1,20 @@
+import type { ObjectId } from 'bson';
```

Comment:
This should be `'../bson'` must've slipped through I'll look into linting for this.